### DISCUSSION TITLE
Feature request: A switch to toggle outline-offset

### DISCUSSION THREAD

**Discussion by chrisxvin** [saadeghi/daisyui#3185] [Ideas / feature requests]
Is it possible to add a switch to toggle the `outline-offset` property? Or add a variable to set the value of it.
This would allow for different design styles.
Thank you.

**Comment by saadeghi** [comment]
What do you mean "switch to toggle the outline-offset"?

**Reply by chrisxvin** [reply]
A variable, set the value of the offset.
If you don't want `outline-offset` (keep the border close to the `input`), you can set it to 0.
![image](https://github.com/user-attachments/assets/8ba40bfe-10e1-4524-bf24-7f94355d5394)

**Reply by saadeghi** [reply]
You can use this class:
```css
focus:outline-offset-0
```
https://play.tailwindcss.com/uS4dunXBLv

You can also apply that class to all text inputs if necessary, or apply it for a theme.

Let me know if you have a question.

**Reply by AimForNaN** [reply]
This should really be variables that we can adjust in `@theme` or something that adjusts the outline for every form element that has an outline. Having to adjust everything either individually or through `@utility` at-rules seems to go against the principle of DaisyUI making development faster, easier and cleaner.

**Reply by pdanpdan** [reply]
It's 3 lines of css to change it however you want for all components you want: https://play.tailwindcss.com/87sOIGvV1N?file=css
